beetsplug/smartplaylist.py

```diff
@@ -28,6 +28,11 @@
 import os
 import six
 
+try:
+    from urllib.request import pathname2url
+except ImportError:
+    from urllib import pathname2url
+
```

Comment:
I suggest adding a comment above to indicate which Python version(s) this compatibility block is intended to support. That will make it easier to find and remove this stanza when Beets drops support for said Python versions(s) in the future.